From 724f10774e51dc27f4b5d91b9963c997a84beea9 Mon Sep 17 00:00:00 2001
From: NghiacoiHN0807 <ytbcointn@gmail.com>
Date: Mon, 19 Jun 2023 23:04:27 +0700
Subject: [PATCH] my changes

---
 src/components/TableXuatXu.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/components/TableXuatXu.js b/src/components/TableXuatXu.js
index 8bc498d..98b1331 100644
--- a/src/components/TableXuatXu.js
+++ b/src/components/TableXuatXu.js
@@ -16,7 +16,7 @@ const TableXuatXu = (props) => {
     setIsShowModalAddNew(false);
     setIsShowModalDelete(false);
   };
-
+  // Show Data On Tables
   useEffect(() => {
     getXanXuat(0);
   }, []);
-- 
2.38.1.windows.1

